ts/Gantt/Tree.ts

```diff
@@ -71,37 +71,40 @@ const getListOfParents = function (
     data: Array<Highcharts.TreePointOptionsObject>,
     ids: Array<string>
 ): Record<string, Array<Highcharts.TreePointOptionsObject>> {
-    const listOfParents = data.reduce(function (
-            prev: (
-                Record<string, Array<Highcharts.TreePointOptionsObject>>
-            ),
+    const listOfParents = data.reduce(
+        function (
+            prev: Record<string, Array<Highcharts.TreePointOptionsObject>>,
             curr: Highcharts.TreePointOptionsObject
         ): Record<string, Array<Highcharts.TreePointOptionsObject>> {
             const parent = pick(curr.parent, '');
 
             if (typeof prev[parent] === 'undefined') {
                 prev[parent] = [];
             }
+
             prev[parent].push(curr);
+
             return prev;
-        }, {} as (
-            Record<string, Array<Highcharts.TreePointOptionsObject>>
-        )),
-        parents = Object.keys(listOfParents);
+        },
+        {} as (Record<string, Array<Highcharts.TreePointOptionsObject>>)
+    );
 
     // If parent does not exist, hoist parent to root of tree.
-    parents.forEach(function (parent: string, list: number): void {
-        const children = listOfParents[parent];
-
-        if ((parent !== '') && (ids.indexOf(parent) === -1)) {
-            children.forEach(function (
-                child: Highcharts.TreePointOptionsObject
-            ): void {
-                (list as any)[''].push(child);
-            });
-            delete (list as any)[parent];
+    const root = '';
+
+    Object.keys(listOfParents).forEach(
+        (node: string): void => {
+            if ((node !== root) && (ids.indexOf(node) === -1)) {
+                listOfParents[root] = [
+                    ...listOfParents[root],
+                    ...listOfParents[node]
+                ];
```

Comment:
`listOfParents[root].push(...listOfParents[node])` [might be marginally quicker](https://www.measurethat.net/Benchmarks/Show/4223/0/array-concat-vs-spread-operator-vs-push#latest_results_block)

Reply:
Cool to know, we will push and not spread in that case! 

On a sidenote, I love that site, but I wish it had a more structured way of browsing it instead of just "the latest" :/